types/account.d.ts

```diff
@@ -218,15 +218,22 @@ export type UpdateCustomerArgs = {
   fullName?: string;
 };
 
-export type Consent = {
-  broadcasterId: number;
+export type ConsentProvider = 'cleeng' | 'jwp';
```

Comment:
do we need a type for the providers?

Reply:
If we find a way to avoid using those values then this type won't be needed.